clients/vscode-hlasmplugin/src/test/syntaxes/endevor/asma.listing.snap

```diff
@@ -313,15 +313,15 @@
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
 >                                                                                                                                    
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
->1077868K allocated to Buffer Pool       Storage required     200K                                                                   
+>1234567B allocated to Buffer Pool       Storage required     200K                                                                   
```

Comment:
I guess it might be a good idea to mask these out as well?

Reply:
I've masked the `1234567B allocated` part. Do you mean that it could be a good idea to mask the `Storage required     200K` part, too?

Reply:
Nevermind... I've just seen a change and did not realize that the values are actually masked already.